pkg/alertmanager/types.go

```diff
@@ -353,6 +353,7 @@ type telegramConfig struct {
 	BotToken             string            `yaml:"bot_token,omitempty" json:"bot_token,omitempty"`
 	BotTokenFile         string            `yaml:"bot_token_file,omitempty" json:"bot_token_file,omitempty"`
 	ChatID               int64             `yaml:"chat_id,omitempty" json:"chat_id,omitempty"`
+	MessageThreadID      *int64            `yaml:"message_thread_id,omitempty" json:"message_thread_id,omitempty"`
```

Comment:
here we want to mimic the native Alertmanager config struct.
```suggestion
	MessageThreadID      int            `yaml:"message_thread_id,omitempty" json:"message_thread_id,omitempty"`
```